site/data/3.11/allMetrics.yaml

```diff
@@ -5436,44 +5453,85 @@
     in the in-memory edge cache. The value should be 0 if compression is not
     used.
 
-- name: rocksdb_cache_edge_effective_entries_size
+- name: rocksdb_cache_edge_empty_inserts_total
+  introducedIn: "3.11.4"
```

Comment:
Will take a while until 3.11.4 gets released. I think we can merge anyway?

Reply:
Yes, we can merge.